### ISSUE TITLE
"unknown world" on startup

### ISSUE THREAD

**Issue by niqofdez** [AerWyn81/AParkour#1]
**Log**: https://pastebin.com/nQ8qPW77
**Parkour**: https://pastebin.com/9bDd2brK

**Multiverse**:
> 21.07 15:56:49 [Server] INFO ====[ Multiverse World List ]====
> 21.07 15:56:49 [Server] INFO world - NORMAL
> 21.07 15:56:49 [Server] INFO world_the_end - THE_END
> 21.07 15:56:49 [Server] INFO world_nether - NETHER
> 21.07 15:56:49 [Server] INFO PVP - NORMAL
> 21.07 15:56:49 [Server] INFO spawnv2 - NORMAL

Maybe an incompatibility with last Multiverse-Core version?

**Comment by AerWyn81** [comment]
Ok I think I understand. I changed a little the behavior of the configuration file loading. In the idea, I added a level to the location to take into account the player's view. So the config goes from :

From  | To
------------- | -------------
![](https://i.imgur.com/9lYdg38.png)| ![](https://i.imgur.com/jQSDZ7B.png)

Either you regenerate the config, or you add a location level for the locations only for the checkpoints and playerPitch/playerYaw.

Normally this should be fine. To be on the safe side and more confident, regenerate the config!
